packages/eslint-plugin-ts/rules/padding-line-between-statements/README.md

```diff
@@ -26,4 +29,17 @@ For example, to add blank lines before interfaces and type definitions:
 }
 ```
 
-**Note:** ESLint `cjs-export` and `cjs-import` statement types are renamed to `exports` and `require` respectively.
+To avoid blank lines between function overloads and the function body:
+
+```jsonc
+{
+  "@stylistic/padding-line-between-statements": [
+    "error",
+    {
+      "blankLine": "never",
+      "prev": "function-overload",
+      "next": "function"
+    }
+  ]
+}
```

Comment:
I think we need to add it to the default options.

Reply:
It might be more readable that way. But that's a personal opinion, so it doesn't matter.

Reply:
We could add it to the default shared config, tho. But IIRC, we didn't enable those rules at all. I guess I was afraid it would be too much and hard to balance well. If we could come up with a solid config to cover majority of the cases, I am happy to have it.